### ISSUE TITLE
can not reproduce the results on MSVD?

### ISSUE THREAD

**Issue by hugddygff** [fanchenyou/HME-VideoQA#2]
Hi, I found that the model provided at 3800 steps has acc 0.317, while I trained by myself and got 0.28.

Are there other tricks?

**Comment by fanchenyou** [comment]
Which method you tested? Is it  _mrm2s? 
The iteration number is not necessary 3800 for each trial. Please run more iterations and pick according to the validation accuracy. Also try finetuning the training by reducing the learning rate after the accuracy stops increasing.

**Comment by hugddygff** [comment]
Yes, I even obtained a higher results...
